posthog/temporal/data_imports/workflow_activities/import_data_sync.py

```diff
@@ -59,12 +60,97 @@ def _trim_source_job_inputs(source: ExternalDataSource) -> None:
         source.save()
 
 
+def _report_heartbeat_timeout(inputs: ImportDataActivityInputs, logger: FilteringBoundLogger) -> None:
+    logger.debug("Checking for heartbeat timeout reporting...")
+
+    try:
+        info = activity.info()
+        heartbeat_timeout = info.heartbeat_timeout
+        current_attempt_scheduled_time = info.current_attempt_scheduled_time
+
+        if not heartbeat_timeout:
+            logger.debug(f"No heartbeat timeout set for this activity: {heartbeat_timeout}")
+            return
+
+        if not current_attempt_scheduled_time:
+            logger.debug(f"No current attempt scheduled time set for this activity: {current_attempt_scheduled_time}")
```

Comment:
nit: Based on the type hints, `current_attempt_scheduled_time` must always be set: https://github.com/temporalio/sdk-python/blob/1.14.2/temporalio/activity.py#L108.

So, not sure if checking for falsy values here makes any sense (at least, believing the type hints).